client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent, useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import { defaultRewindConfig, RewindConfig } from './types';
+import { rewindBackup } from 'state/activity-log/actions';
+import { useLocalizedMoment } from 'components/localized-moment';
+import CheckYourEmail from './rewind-flow-notice/check-your-email';
+import RewindFlowNotice, { RewindFlowNoticeLevel } from './rewind-flow-notice';
+import getBackupDownloadId from 'state/selectors/get-backup-download-id';
+import getBackupDownloadUrl from 'state/selectors/get-backup-download-url';
+import getBackupDownloadProgress from 'state/selectors/get-backup-download-progress';
+import ProgressBar from './progress-bar';
+import QueryRewindBackupStatus from 'components/data/query-rewind-backup-status';
+import RewindConfigEditor from './rewind-config-editor';
+
+interface Props {
+	rewindId: string;
+	siteId: number;
+}
+
+const BackupDownloadFlow: FunctionComponent< Props > = ( { rewindId, siteId } ) => {
+	const dispatch = useDispatch();
+	const moment = useLocalizedMoment();
+	const translate = useTranslate();
+
+	const [ rewindConfig, setRewindConfig ] = useState< RewindConfig >( defaultRewindConfig );
+
+	const downloadId = useSelector( state => getBackupDownloadId( state, siteId, rewindId ) );
+	const downloadUrl = useSelector( state => getBackupDownloadUrl( state, siteId, rewindId ) );
+	const downloadProgress = useSelector( state =>
+		getBackupDownloadProgress( state, siteId, rewindId )
+	);
+
+	const downloadTimestamp = moment.unix( rewindId ).format( 'LLL' );
+
+	const requestDownload = useCallback(
+		() => dispatch( rewindBackup( siteId, rewindId, rewindConfig ) ),
+		[ dispatch, rewindConfig, rewindId, siteId ]
+	);
+
+	const renderConfirm = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-ready.svg"
+					alt="jetpack cloud download ready"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Create downloadable backup' ) }</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'{{strong}}%(downloadTimestamp)s{{/strong}} is the selected point to create a download backup of. ',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( '{{strong}}%(downloadTimestamp)s{{/strong}} is the selected point to create a download backup of.' )` *ES Score: 8*